schemas/com.snowplowanalytics.snowplow.media/ad_break/jsonschema/1-0-0

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity, shared with all media_player_ad events belonging to the ad break.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.media",
+    "name": "ad_break",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "name": {
+      "description": "Ad break name (e.g., pre-roll, mid-roll, and post-roll).",
+      "type": [
+        "null",
+        "string"
+      ],
+      "maxLength": 4096
+    },
+    "breakId": {
+      "type": "string",
+      "maxLength": 256,
+      "description": "An identifier for the ad break."
+    },
+    "startTime": {
+      "type": "number",
+      "description": "Playback time in seconds at the start of the ad break.",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "breakType": {
+      "description": "Type of ads within the break: linear (take full control of the video for a period of time), nonlinear (run concurrently to the video), companion (accompany the video but placed outside the player).",
+      "enum": [
```

Comment:
@jethron Thoughts on removing the enum here? There may be other things we otherwise need to include (overlays - complete or partial, outstream (not sure if we should include these at all?).

Reply:
IMO overlays is a special case of companion and outstream is still linear; the context of the player's location in the page isn't captured in this property so I don't think should be factored in here. It _might_ be useful for users to populate this freeform, but probably only worth the flexibility if we don't intend specific handling of the value in the modeling stage.